pkgs/tools/security/prs/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, pkg-config
+, python3
+, dbus
+, glib
+, gnupg
+, gtk3
+, libxcb
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "prs";
+  version = "0.2.1";
+
+  src = fetchFromGitLab {
+    owner = "timvisee";
+    repo = "prs";
+    rev = "v${version}";
+    sha256 = "0nicvcc94ks7mvbjjf0iyym952v5c1wifwjfh7bq5rf7kswj3j0p";
+  };
+
+  cargoSha256 = "1rrlzy55yvfidica0v36rhn5yigc7aj3fc18dmmsv203v91n6c25";
+
+  #cargoBuildFlags = [ "--features" "backend-gpgme" ];
```

Comment:
I get
```
error: --features is not allowed in the root of a virtual workspace
```
if I eneble this.

Reply:
cc @andir @danieldk @zowoq

Reply:
This is an annoying property of Cargo workspaces. You'd need to build in the individual directories that have this feature. Since using `backend-gpgme` and `backend-gnupg-bin` together leads to unreachable code (and perhaps other problems), you also have to disable the default features and then list all the other features besides `backend-gpgme`.

I think in this case it would be easier to do a `substituteInPlace` on `cli/Cargo.toml` and `gtk3/Cargo.toml` and replace `"backend-gnupg-bin"` by `"backend-gpgme"` to change the default features.

Reply:
> I think in this case it would be easier to do a `substituteInPlace` on `cli/Cargo.toml` and `gtk3/Cargo.toml` and replace `"backend-gnupg-bin"` by `"backend-gpgme"` to change the default features.

Thanks a lot! I'll do that.